controllers/helper.go

```diff
@@ -62,3 +78,156 @@ func GetDefaultNodeSelector() map[string]string {
 	return map[string]string{"node-role.kubernetes.io/worker": "",
 		"kubernetes.io/os": "linux"}
 }
+
+func syncPluginDaemonObjs(ctx context.Context, client k8sclient.Client, scheme *runtime.Scheme, dp *sriovnetworkv1.SriovNetworkNodePolicy, pl *sriovnetworkv1.SriovNetworkNodePolicyList) error {
+	logger := log.Log.WithName("syncPluginDaemonObjs")
+	logger.Info("Start to sync sriov daemons objects")
```

Comment:
please, reduce verbosity
```suggestion
	logger.V(1).Info("Start to sync sriov daemons objects")
```

also on L155 and L184

Reply:
done